http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyConfiguration.java

```diff
@@ -53,10 +53,8 @@ public int connectionTimeout() {
     /**
      * @see NettySdkHttpClientFactory.Builder#maxConnectionsPerEndpoint(Integer)
      */
-    @ReviewBeforeRelease("Does it make sense to use this value? Netty's implementation is max connections" +
```

Comment:
Has this been resolved? Are we happy that `MAX_CONNECTIONS` means different things between Netty and Apache?

Reply:
I think so, my reasoning for removing it was that the value is only present in two scenarios.

1) A default from the service or global defaults. If this is being applied the customer is using the default HTTP client or passing in a factory without maxConnectionsPerEndpoint set. Either way the client is used only for that service so it has the same meaning as just MAX_CONNECTIONS.
2) The customer passes in a custom http client or custom http factory with max connections per endpoint set. Here they are explicitly setting the netty property on the builder and are fully aware of what it means.

I guess there is a third case where they create their own http client or factory without setting the max connections per endpoint in which case we have to default to something and the global max conns seems just as good as anything else.